engine/src/main/java/io/camunda/zeebe/engine/processing/usertask/UserTaskCommandPreconditionChecker.java

```diff
@@ -22,6 +22,12 @@ public class UserTaskCommandPreconditionChecker {
       "Expected to %s user task with key '%d', but no such user task was found";
   private static final String INVALID_USER_TASK_STATE_MESSAGE =
       "Expected to %s user task with key '%d', but it is in state '%s'";
+  private static final String INVALID_USER_TASK_ASSIGNEE_MESSAGE =
+      "Expected to %s user task with key '%d', but it has already been assigned";
+  private static final String INVALID_USER_TASK_EMPTY_ASSIGNEE_MESSAGE =
+      "Expected to %s user task with key '%d', but the assignee is empty";
```

Comment:
A new message when an empty string is passed as an assignee. Not entirely happy with this one, the suggestions are welcome @koevskinikola🙂